### ISSUE TITLE
Chat bot to have a conversation with the teacher

### ISSUE THREAD

**Issue by sujaysathya** [sujaysathya/bunk_bot#5]
Building a chatbot which contextually replies to some basic/regular questions asked by a teacher

**Comment by denismaxim0v** [comment]
Dialog flow?

**Comment by devashar13** [comment]
Does the voice recognition create a live transcript?

**Comment by Sandeep-BlackHat** [comment]
yes, but we have do intense research for that, but it's a great idea!

**Comment by danyyboi** [comment]
how to use this bro

**Comment by sujaysathya** [comment]
youtube tutorial has been added

**Comment by Sandeep-BlackHat** [comment]
link??

**Comment by dhiganthrao** [comment]
@Sandeep-BlackHat check README.

**Comment by rivetin** [comment]
> Dialogflow?

Yeah, it is a great choice but the built-in greeting intent (you can create as much as your heart's content)must be tweaked to match a particular dialect. But even after all those tweaking at a point in time we'll realize that the google AI works best when it is given the dialect some ting similar to the US or maybe even the UK. the more it fails in picking the context of the chat the more it is redirected to default Fallback Intent (If no other fallback intents are defined )

**Comment by wrecker3000** [comment]
please make this bot for zoom also.

**Comment by dijorajsenroy** [comment]
how about a RASA chatbot instead?
Great project btw.

**Comment by viper-adnan** [comment]
How about Lydia by coffeehouse ?
